### PR TITLE
chore: create SECURITY.md

### PR DESCRIPTION
**Pull request by BoppLi**
## Description

Github [recommends](https://github.com/SchwarzIT/onyx/community) to provide information in regards to vulnerability handling.

## Checklist

- [x] The added / edited code has been documented with [JSDoc](https://jsdoc.app/about-getting-started)
- [x] All changes are documented in the documentation app (folder `apps/docs`)
- [x] If a new component is added, at least one [Playwright screenshot test](https://github.com/SchwarzIT/onyx/actions/workflows/playwright-screenshots.yml) is added
- [x] A changeset is added with `npx changeset add` if your changes should be released as npm package (because they affect the library usage)

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Security Policy
+
+## Supported Versions
+
+The following versions of onyx are currently being supported with security updates.
```
Comment (JoCa96) [inline SECURITY.md:5]: If we do it like this we should add a section for each individuial package that we publish.
But as we are currently in the alpha stage for all package we might just want to tell people that we are in development with everything and update this file on the version 1.0.0 release.

**Review by JoCa96 — CHANGES_REQUESTED** [review]
wait for open topics

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 0.0.0   | :white_check_mark: |
```
Comment (BoppLi) [inline SECURITY.md:9]: Onyx is currently still in alpha development. Once we published 1.0.0 of our packages, we'll show the information on wich versions of which packages are being supported with security updates below.

File: SECURITY.md (hunk 1 above)
Comment (BoppLi) [reply SECURITY.md]: Changed the text, is it ok like this? :)

**Review by larsrickert — APPROVED** [review]

**Review by JoCa96 — APPROVED** [review]

**Pull request merged**